module/core/src/Application/Normalizer/NormalizerInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Core\Application\Normalizer;
+
+use JMS\Serializer\DeserializationContext;
+use JMS\Serializer\SerializationContext;
+
+interface NormalizerInterface
+{
+    public function normalize(object $data, ?SerializationContext $context, ?string $type): array;
```

Comment:
You can't use JMS Context on ergonode Interface this connect it to JMS serizlaizer.